src/main/java/core/basesyntax/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculateArea {
```

Comment:
Verbs are suitable for methods naming. Interfaces are nouns or adjectives
```suggestion
public interface AreaCalculator {
```